src/Templates/src/templates/maui-lib/MauiLib1.csproj

```diff
@@ -1,16 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>DOTNET_TFM_VALUE;DOTNET_TFM_VALUE-android;DOTNET_TFM_VALUE-ios;DOTNET_TFM_VALUE-maccatalyst</TargetFrameworks>
-		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) and '$(MSBuildRuntimeType)' == 'Full'">$(TargetFrameworks);DOTNET_TFM_VALUE-windows10.0.19041</TargetFrameworks>
+		<TargetFrameworks>DOTNET_TFM;DOTNET_TFM-android;DOTNET_TFM-ios;DOTNET_TFM-maccatalyst</TargetFrameworks>
```

Comment:
Why this change? I didn't see a reasoning in the description, and I'm not sure how it is connected to the slash issue. Is it just making the parameter naming consistent?

Reply:
Ah, this was unifying all the string substitutions to be in one file. It was already done before by editing template.json _and_ editing the csproj. Now it all goes in template.json and gets substituted using the .NET CLI template engine.